src/rdkafka_cgrp.c

```diff
@@ -6484,6 +6533,9 @@ static rd_kafka_op_res_t rd_kafka_cgrp_op_serve(rd_kafka_t *rk,
                 rko = NULL;
                 break;
 
+        /**
+         * TODO: Check regex for KIP-848
+         */
```

Comment:
This returns the subscription so if we remove it from `rkcg_subscription` it can return an empty subscription if we have all regexes. We can have two new fields: `rkcg_subscription_topics` and the existing `rkcg_subscription_regex`.  `rkcg_subscription` remains unchanged and `rkcg_subscription_topics` is the same with the regexes removed.